### PR TITLE
Fixes and added Things

### PR DESCRIPTION
**Pull request by jani270**
- Added Snowglobe Skin
- Added Wiki URL for Hyperion and Valkyrie
- Added Recipe for Catacombs Expert Ring
- Fixed Sheep Pets and Bat Pet (WeirdChamp Hypixel Wiki)

Merry Christmas 

_hope all things are valid_

### CODE REVIEW

File: items/SHEEP;0.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "§8Alchemy Pet",
     "",
     "§7Intelligence: §a{INTELLIGENCE}",
-    "§7Ability Damage: §a{ABILITY_DAMAGE}",
+    "§7Ability Damage: §a{ABILITY_DAMAGE}%",
```
Comment (IRONM00N) [inline items/SHEEP;0.json:10]: ?

**Comment by IRONM00N** [comment]
Not Sure why you added `%` but other than that it looks good

**Comment by jani270** [comment]
> Not Sure why you added `%` but other than that it looks good

http://prntscr.com/w9atx3

**Comment by IRONM00N** [comment]
okay looks good then

**Pull request merged**